### ISSUE TITLE
can i get the softmax tensor before argmax

### ISSUE THREAD

**Issue by birdflyto** [open-mmlab/mmsegmentation#121]
hello:
Thanks for your great job. But i have a question about how can we get  the softmax tensor before argmax or other mid logits tensor with your pipline?
Hopefully,you can give some tips,
Thanks

**Comment by xvjiarui** [comment]
Hi @birdflyto 
You need to add softmax in the model. Which stage would you like to apply softmax? Train or test?

**Comment by birdflyto** [comment]
Hi @xvjiarui ,maybe both and the other mid output to be saved  , cause it seems the model is packaged in the pipline 
i am not sure the add softmax operation is in mmsegmentation/mmseg/models/segmentors/encoder_decoder.py /inference or or other place. Could you give a more detail guid?
thanks

**Comment by xvjiarui** [comment]
Hi @birdflyto 
You may consider somewhere around [this line](https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/segmentors/encoder_decoder.py#L264).

**Comment by birdflyto** [comment]
Hi @xvjiarui 
Thank you very much
